lighthouse-core/gather/driver/execution-context.js

```diff
@@ -51,6 +52,9 @@ class ExecutionContext {
   async _getOrCreateIsolatedContextId() {
     if (typeof this._executionContextId === 'number') return this._executionContextId;
 
+    await this._session.sendCommand('Page.enable');
+    await this._session.sendCommand('Runtime.enable');
```

Comment:
before this dependency was an implicit global relationship, felt better to make it explicit.